crates/nu-command/src/system/ps.rs

```diff
@@ -127,13 +135,11 @@ fn run_ps(engine_state: &EngineState, call: &Call) -> Result<PipelineData, Shell
                         Vec::new(),
                     )
                 })?;
-                let proc_start = match proc_stat.starttime() {
-                    Ok(t) => t,
-                    Err(_) => {
-                        // If we can't get the start time, just use the current time
-                        chrono::Local::now()
-                    }
-                };
+                // If we can't get the start time, just use the current time
+                let proc_start = proc_stat
+                    .starttime()
+                    .get()
+                    .unwrap_or_else(|_| chrono::Local::now());
```

Comment:
With all the feature gating unsure if this else branch is ever reached but we may want to reconsider this implicit behavior. We could return `null` instead